ServiceWebsite/ServiceWebsite.AcceptanceTests/TestClients/BookingsApiTestClientTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using NUnit.Framework;
```

Comment:
I dont think NUnit runs tests in sequence, so these test will probably fail sometimes because of inconstant data

Reply:
the tests are independent from each other

Reply:
yep each test is independent. The data cleanup and setup runs before EVERY test.